app/dashboard/templates/bounty/change.html

```diff
@@ -0,0 +1,146 @@
+{% comment %}
+  Copyright (C) 2018 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  {% include 'shared/head.html' %}
+  {% include 'shared/cards.html' %}
+  <link rel="stylesheet" href={% static "v2/css/submit_bounty.css" %}>
+</head>
+
+<body class="interior submit_bounty" style="background-color: #FFFFFF;">
+  {% include 'shared/tag_manager_2.html' %}
```

Comment:
why build a whole new form. why not just re-use the `new_bounty` form?

Reply:
I take a look at the new changes and modify the template if necessary for easier reuse. 👍